cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,44 +1,53 @@
-import Image from "next/image"
+import Image, { StaticImageData } from "next/image"
 import Link from "next/link"
 import generateImageUrl from "@/helpers/imageURLHelper"
 import Categories from "@/types/models/categories"
-import { ClientCvModel } from "@/types/models/cv"
+import { useState } from "react"
+import blankCvImg from "../../../public/images/blackCv.png";
```

Comment:
Fix import stylel (should be fixed with prettier)

Reply:
I'll update that but still, this file is outdated